openhtf/exe/__init__.py

```diff
@@ -36,6 +36,11 @@ class TestStopError(Exception):
   """Test is being stopped."""
 
 
+def _InvalidTestStart():
```

Comment:
Alright, now no test_start just leaves the dut_id as None. If they don't set the dut_id, they'll get some weird behavior later.